decidim/decidim
# Fix Video embeds are not shown in short_description field

#### :tophat: What? Why?
This PR tries to add more consistency on displaying HTML input tags. 
One one hand we filtering more elements (form related fields, iframes, images, and other related media) when displaying the data that has been input by the end user. 
On the other hand we add more consistency on displaying the admin related elements.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #10673
- Fixes #10130

#### Testing
1. Login to admin area, on the participatory process
2. Add some image and video to the short description
3. Go to public website, and see there is no video displayed
4. Apply patch 
5. See the video is being rendered

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
